en/sync-themes-plugin.md

```diff
@@ -111,8 +111,8 @@ Instead of publishing a new prototype, you may want to update an existing one by
 
 Once the prototype is successfully published to the cloud, you will be handed the link to it plus options to:
 
-- view it in the workspace it belongs to, which will opent it up in a browser window
-- create usability study, which will initialize a [remote usability study](prototyping/setting-up-a-usability-study.md) and open it in a browser window for further configuration
+- view it in the workspace it belongs to, which will open it up in a browser window
+- create usability study, which will initialize a remote [usability study](prototyping/set-up-a-user-test.md) and open it in a browser window for further configuration
```

Comment:
```suggestion
- create usability study, which will initialize a [user test](prototyping/set-up-a-user-test.md) and open it in a browser window for further configuration
```

Reply:
Usability Study was kept there because if was the actual button displays.
@ig-georgeA could you confirm?

Reply:
@pmoleri sorry. It should say ..."Create a usability test"

Reply:
@ig-georgeA 
like this?
create a usability test, which will initialize a remote [user test]
see above the suggested change too and accept it if it is OK